### ISSUE TITLE
Using core.cache instead of a cache folder?

### ISSUE THREAD

**Issue by Jared314** [edgecase/dieter#57]
I'm using memcache (through the Spyglass core.cache implementation) to store rendered items between frontends, but dieter is storing assets on the local file system. I would love to unify that storage.

Are there any plans to abstract the asset caching mechanisms?

**Comment by pbiggar** [comment]
Yes, actually. I've been working on a complete rewrite that supports this better: the idea is that dieter would still compile to the file system, but that it wouldn't actually serve the assets, they'd be served through a CDN, or possibly through some other mechanism like this.

It's a different project because it's not compatible, and breaks a lot of a lot of assumptions in dieter. It's not done yet, but it's maybe 95% of the way there: https://github.com/circleci/stefon.

**Comment by vbauer** [comment]
It would be great. :+1:
